### DISCUSSION TITLE
Rubrica Final Proyecto Grupo1 : Auto

### DISCUSSION THREAD

**Discussion by Gona79** [ISPC-TST-EM2022/Grupo1#92] [Announcements]
Nota de la 1er Semana: 6 (seis)

Que tal estimados, el video que hicieron esta muy bien, pero esta semana debian: 
1) Diagramar el proyecto
2) Perfilar el proyecto temporalmente con un diagrama gantt
3) Completar los formularios que subio el profe Jorge Morales @JorEl057 

Aun asi seguramente no van a tener inconvenientes en avanzar apropiadamente, saludos.

**Comment by Gona79** [comment]
Nota de 3er Semana:  10 (diez)

Chicos el objetivo del desarrollo del proyecto era conocer las herramientas de:
Creacion de proyectos ( Gantt, AGILE, Proyectos)
Ideacion, desarrollo y prueba. 
Pero principalmente el trabajo colaborativo. 
Lo hicieron muy bien, adapten este trabajo en forma de plantilla y les sera util para el resto de los proyectos de la cursada. Continuen asi.

**Reply by GiseRomero** [reply]
Muchas gracias profe!

**Comment by marlufer** [comment]
Muchas Graxias Profe!
